### PR TITLE
Add documentation for configuring alerts using Alertmanager

### PR DESCRIPTION
**Pull request by UtkarshBhatthere**
Add MicroCeph how-to doc for configuration of alerts using Alertsmanager

### CODE REVIEW

File: docs/how-to/alerts.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
+Enable Alerts on MicroCeph using Prometheus Alertmanager
+========================================================
+
+Pre-Requisite
+-------------
+In order to configure Alerts, your MicroCeph deployment must enable metrics collections with Prometheus. Follow :doc:`this How-To <metrics>` if you haven't configured it. Also, Alertmanager is distributed as a separate binary which should be installed and running.
```
Comment (sabaini) [inline docs/how-to/alerts.rst:6]: s/Alerts/alerts

File: docs/how-to/alerts.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+..  code-block:: none
+
+    prometheus --config.file=microceph.yaml
+
+Click on the 'Alerts' tab on Prometheus dashboard to view the configured alerts:
+
+.. figure:: assets/alerts
+
+Look we already have an active alert! which means that Alertmanager is configured and working.
```
Comment (sabaini) [inline docs/how-to/alerts.rst:61]: Possibly add some context about what we're seeing just to be extra-clear, something like

> As an example, the screenshot shows an active CephHealthWarning, while all other configured alerts are inactive

Comment (UtkarshBhatthere) [reply docs/how-to/alerts.rst]: Ack

File: docs/how-to/alerts.rst (hunk 1 above)
Comment (UtkarshBhatthere) [reply docs/how-to/alerts.rst]: ack

**Comment by UtkarshBhatthere** [comment]
Merging PR post comment resolution.

**Pull request merged**